utils/default-helpers/src/main/scala/pl/touk/nussknacker/engine/util/functions/conversion.scala

```diff
@@ -20,4 +22,116 @@ trait ConversionUtils extends HideToString {
+  }
+
+  @Documentation(description = "Convert any value to Boolean or throw exception in case of failure")
+  def toBooleanOrNull(@ParamName("value") value: Any): java.lang.Boolean = value match {
+    case v: String            => java.lang.Boolean.valueOf(v)
+    case v: java.lang.Boolean => v
+    case _                    => null
+  }
+
+  @Documentation(description = "Convert any value to Integer or throw exception in case of failure")
+  def toInteger(@ParamName("value") value: Any): java.lang.Integer = value match {
+    case v: String => Integer.valueOf(numeric.toNumber(v).intValue())
+    case v: Number => v.intValue()
+    case null      => null
+    case _         => throw new IllegalArgumentException(s"Cannot convert: $value to Integer")
+  }
+
+  @Documentation(description = "Convert any value to Integer or null in case of failure")
+  def toIntegerOrNull(@ParamName("value") value: Any): java.lang.Integer = value match {
+    case v: String => Try(Integer.valueOf(numeric.toNumber(v).intValue())).getOrElse(null)
+    case v: Number => v.intValue()
+    case _         => null
+  }
+
+  @Documentation(description = "Convert any value to Long or throw exception in case of failure")
+  def toLong(@ParamName("value") value: Any): java.lang.Long = value match {
+    case v: String => java.lang.Long.valueOf(numeric.toNumber(v).longValue())
+    case v: Number => v.longValue()
+    case null      => null
+    case _         => throw new IllegalArgumentException(s"Cannot convert: $value to Long")
+  }
+
+  @Documentation(description = "Convert any value to Long or null in case of failure")
+  def toLongOrNull(@ParamName("value") value: Any): java.lang.Long = value match {
+    case v: String => Try(java.lang.Long.valueOf(numeric.toNumber(v).longValue())).getOrElse(null)
+    case v: Number => v.longValue()
+    case _         => null
+  }
+
+  @Documentation(description = "Convert any value to Double or throw exception in case of failure")
+  def toDouble(@ParamName("value") value: Any): java.lang.Double = value match {
+    case v: String => java.lang.Double.valueOf(numeric.toNumber(v).doubleValue())
+    case v: Number => v.doubleValue()
+    case null      => null
+    case _         => throw new IllegalArgumentException(s"Cannot convert: $value to Long")
+  }
+
+  @Documentation(description = "Convert any value to Double or null in case of failure")
+  def toDoubleOrNull(@ParamName("value") value: Any): java.lang.Double = value match {
+    case v: String => Try(java.lang.Double.valueOf(numeric.toNumber(v).doubleValue())).getOrElse(null)
+    case v: Number => v.doubleValue()
+    case _         => null
+  }
+
+  @Documentation(description = "Convert any value to BigInteger or throw exception in case of failure")
+  def toBigInteger(@ParamName("value") value: Any): java.math.BigInteger = value match {
+    case v: String               => new java.math.BigInteger(v)
```

Comment:
I'm not sure if we should rely on this BigInteger constructor. Maybe it's better to use the `numeric` util also here?

Reply:
Done

Reply:
Can we do the same in case of BigDecimal constructor?

Reply:
Yes, but I would rather will not do it because in numeric we try to parse a double at first try and I am not sure if it's ok to do it in a such order.